.run/router dev.run.xml

```diff
@@ -2,9 +2,9 @@
   <configuration default="false" name="router dev" type="GoApplicationRunConfiguration" factoryName="Go Application">
     <module name="cosmo" />
     <working_directory value="$PROJECT_DIR$/router" />
-    <go_parameters value="-race" />
```

Comment:
Are both this subtraction and addition intended?

Reply:
No, thanks.